stdlib/public/CommandLineSupport/CommandLine.cpp

```diff
@@ -124,25 +124,27 @@ char **_swift_stdlib_getUnsafeArgvArgc(int *outArgLen) {
     if (szBufferSize == 0) {
       swift::fatalError(0,
                         "Fatal error: Could not retrieve commandline "
-                        "arguments: %u\n",
-                        GetLastError());
+                        "argument %d: %lu\n",
+                        i, static_cast<unsigned long>(GetLastError()));
       return nullptr;
     }
 
     char *buffer = static_cast<char *>(
         calloc(static_cast<size_t>(szBufferSize), sizeof(char)));
     if (buffer == nullptr) {
       swift::fatalError(0,
-                        "Fatal error: Could not allocate space for commandline"
-                        "arguments");
+                        "Fatal error: Could not allocate space for commandline "
+                        "argument %d: %d\n",
+                        i, errno);
```

Comment:
`errno` will pretty much be `ENOMEM` no?

Reply:
Or `EINVAL` or `ENOSYS` or `EPERM` or…

Reply:
(But yes, in practice we can expect it to always be `ENOMEM`. This is an unexpected failure either way.)

Reply:
It was more to make sure I understood what the failure modes were that we should/could expect :).  I agree with the change in principle.